src/document/DocumentCommandHandlers.js

```diff
@@ -159,8 +168,11 @@ define(function (require, exports, module) {
             windowTitle         = brackets.config.app_title,
             currentlyViewedFile = MainViewManager.getCurrentlyViewedFile(MainViewManager.ACTIVE_PANE),
             currentlyViewedPath = currentlyViewedFile.fullPath,
-            readOnlyString      = currentlyViewedFile.readOnly ? "[Read Only] - " : "";
-
+            readOnlyString      = currentlyViewedFile.readOnly ? "[Read Only] - " : "",
+            filepathDisplayPref = PreferencesManager.get('showFullFilePath');
```

Comment:
Variable name should follow the naming convention. 'p' in path should be capital.